wp-content/themes/heideandcook/src/slick/slick/slick-theme.css

```diff
@@ -36,7 +36,7 @@
 
     cursor: pointer;
 
-    color: transparent;
+    background: #006940;
```

Comment:
You shouldn't be editing this file since it a part of the slick lib. Instead overwrite the styles in your sass.